test/model/test/common/wiredtiger_util.cpp

```diff
@@ -32,6 +32,7 @@
 #include "wiredtiger.h"
 extern "C" {
 #include "test_util.h"
+#include "../../../../src/log/log_private.h"
```

Comment:
This is ugly but necessary for the moment. It can be cleaned up as part of WT-13610

Reply:
sgtm